x-pack/plugins/security_solution/public/common/components/ml/tables/anomalies_user_table.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useAnomaliesTableData } from '../anomaly/use_anomalies_table_data';
+import { HeaderSection } from '../../header_section';
+
+import { hasMlUserPermissions } from '../../../../../common/machine_learning/has_ml_user_permissions';
+import * as i18n from './translations';
+
+import { Loader } from '../../loader';
+import { AnomaliesUserTableProps } from '../types';
+import { useMlCapabilities } from '../hooks/use_ml_capabilities';
+import { BasicTable } from './basic_table';
+
+import { getCriteriaFromUsersType } from '../criteria/get_criteria_from_users_type';
+import { Panel } from '../../panel';
+import { anomaliesTableDefaultEquality } from './default_equality';
+import { convertAnomaliesToUsers } from './convert_anomalies_to_users';
+import { getAnomaliesUserTableColumnsCurated } from './get_anomalies_user_table_columns';
+
+const sorting = {
+  sort: {
+    field: 'anomaly.severity',
+    direction: 'desc',
+  },
+} as const;
+
+const AnomaliesUserTableComponent: React.FC<AnomaliesUserTableProps> = ({
+  startDate,
+  endDate,
+  userName,
+  skip,
+  type,
+}) => {
+  const capabilities = useMlCapabilities();
+
+  const [loading, tableData] = useAnomaliesTableData({
+    startDate,
+    endDate,
+    skip,
+    criteriaFields: getCriteriaFromUsersType(type, userName),
+    filterQuery: {
+      exists: { field: 'user.name' },
```

Comment:
@elastic/ml-ui I have to display anomalies related to users. So I am only returning anomalies where `user.name` exists. Can I do that?

Reply:
This will only work if `user.name` is included as one of the influencer fields for the job. Is `user.name` included inside the `influencers` array inside the `analysis_config` for the job(s)?

Reply:
I believe that `user.name` should be there for all user anomalies. 
I checked many auth ml configurations and `user.name` was there for all of them. For example: 

`windows_rare_user_runas_event.json`
```
  "analysis_config": {
    "bucket_span": "15m",
    "detectors": [
      {
        "detector_description": "rare by \"user.name\"",
        "function": "rare",
        "by_field_name": "user.name"
      }
    ],
    "influencers": [
      "host.name",
      "process.name",
      "user.name"
    ]
  },
```